MapboxNavigation/SystemSpeechSynthesizer.swift

```diff
@@ -5,7 +5,7 @@ import MapboxCoreNavigation
 import MapboxSpeech
 
 /**
- `SpeechSynthesizing` implementation, using `AVSpeechSynthesizer`. Supports only english language.
+ `SpeechSynthesizing` implementation, using `AVSpeechSynthesizer`. Supports only English language.
```

Comment:
This implementation seems to support more than just English now. I don’t think it was ever the intention to have it support only English. Only the Alex voice is English-specific.